.github/workflows/lint-and-test.yml

```diff
@@ -7,7 +7,7 @@ jobs:
 
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-latest, windows-latest]
+        os: [ubuntu-latest, macos-10.15, windows-latest]  # macos-latest does not have py3.6
```

Comment:
This has nothing to do with the pandas change; it's because `macos-latest` recently changed to macos 11 instead of macos 10, and they dropped python 3.6 in macos 11.  That made the 3.6 test fail here because this is the first PR since the change.  See the table here: https://github.com/actions/virtual-environments/issues/4060.

We might consider dropping python 3.6 from testing though as it is slated to reach EOL in a week or two: https://devguide.python.org/#status-of-python-branches

Reply:
+1 to dropping support for python 3.6. I'm OK to do that in this PR if it simplifies the pandas requirement.

Reply:
I don't think dropping 3.6 changes anything for the pandas requirement.  pvanalytics's current minimum (0.24.0) supports python 3.7, so dropping 3.6 is not itself an excuse to bump our minimum and skip over the problematic 1.1.x series.